lib/datadog/tracing/contrib/sidekiq/server_tracer.rb

```diff
@@ -68,10 +76,16 @@ def call(worker, job, queue)
 
               yield
             end
+            # rubocop:enable Metrics/MethodLength
+            # rubocop:enable Metrics/AbcSize
           end
 
           private
 
+          def propagation
+            Sidekiq::Distributed::Propagation::INSTANCE
```

Comment:
Use `@propagation = Sidekiq::Distributed::Propagation.new` in the `initializer` instead.